train/tictactoe.py

```diff
@@ -46,14 +46,26 @@ def _get_input_shape(self, cfg):
         return shape_cpu if cfg["cpu"] else shape_gpu
 
     def _create_model_simple_two_headed(self, cfg):
+        l2reg = cfg["model"]["l2reg"]
```

Comment:
l2reg = tf.keras.regularizers.l2(l=cfg["model"]["l2reg"])

probably does the same, but lets be explicit about using L2 regularization

Reply:
done

Reply:
done